packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx

```diff
@@ -53,79 +71,102 @@ export const NotificationsWidget: FC<NotificationsWidgetProps> = ({ ...rest }) =
-                            markNotificationsAsRead(notification)
-                          }}
-                        />
-                      ))}
-                    </div>,
-                  ],
-                }}
-                footerProps={{
-                  type: 'infinite',
-                  reachedEnd: !pageInfo?.hasNextPage ?? true,
-                  fetchMore: async () => {
-                    await fetchMore({
-                      variables: {
-                        after: pageInfo?.endCursor,
-                        first: 10,
-                      },
-                      updateQuery: (prev, { fetchMoreResult }) => {
-                        fetchMoreResult.notificationsConnection.edges = [
-                          ...(prev.notificationsConnection?.edges ?? []),
-                          ...fetchMoreResult.notificationsConnection.edges,
-                        ]
-                        return fetchMoreResult
+                    {
+                      label: `${isMemberType ? 'Member' : 'Channel'} notification center`,
+                      nodeStart: <SvgActionNotifications />,
+                      onClick: popoverRef.current?.hide,
+                      to: isMemberType
+                        ? absoluteRoutes.viewer.memberNotifications()
+                        : absoluteRoutes.studio.channelNotifications(),
+                    },
+                    {
+                      label: `${isMemberType ? 'Member' : 'Channel'} notification settings`,
+                      nodeStart: <SvgActionSettings />,
+                      onClick: () => markNotificationsAsRead(notifications),
+                    },
+                  ]}
+                  trigger={null}
+                  triggerTarget={ref}
+                />
+              </div>
+            </Header>
+            <Content>
+              {notifications.length > 0 ? (
+                <Section
+                  withoutGap
+                  contentProps={{
+                    type: 'grid',
+                    grid: {
+                      sm: {
+                        columns: 1,
                       },
-                    })
-                  },
-                }}
-              />
-            ) : (
-              <EmptyFallback variant="small" title="You don't have any notifications" />
-            )}
-          </Content>
-          <StyledButton
-            variant="tertiary"
-            size="large"
-            icon={<SvgActionNotifications />}
-            fullWidth
-            to={absoluteRoutes.viewer.notifications()}
-            onClick={popoverRef.current?.hide}
-          >
-            <Text as="span" variant="t100">
-              Go to notification center
-            </Text>
-          </StyledButton>
-        </Wrapper>
+                    },
+                    children: [
+                      <div key="single">
+                        {notifications.map((notification, idx) => (
```

Comment:
probably needs a fallback similar to `NotificationsViewContent.tsx` for the notification with no id?